libs/ui/src/election_info_bar.tsx

```diff
@@ -48,9 +43,15 @@ export function ElectionInfoBar({
         <NoWrap as="strong">{electionDefinition.election.title}</NoWrap> —{' '}
         <NoWrap>{electionDate}</NoWrap>
         <Text as="div" small>
-          {showPrecinctInfo && (
+          {precinctSelection && (
             <React.Fragment>
-              <NoWrap>{precinct?.name ?? 'All Precincts'}</NoWrap> —{' '}
+              <NoWrap>
+                {getPrecinctSelectionName(
```

Comment:
A slightly more intuitive name for this helper might be `getSelectedPrecinctName` or `precinctSelectionToReadableString`

Reply:
The current name makes the most sense to me, as `Selected` feels like it implies a selection was made when that was not always the case and `ReadableString` is less clear than `Name` only because `Name` is how we've been identifying these strings in code to date.

Reply:
Ah good point that the precinct isn't actually selected on VxCentralScan - the current name is okay then!